addons/kibana/6.8.x/kibana-8.yaml

```diff
@@ -59,9 +59,7 @@ spec:
         requests:
           cpu: 100m
       plugins:
-        # to avoid needing to download any plugins at runtime, use a container and a shared volume
-        # do not enable the plugins here, instead rebuild the mesosphere/kibana-plugins image with the new plugins
-        enabled: false
```

Comment:
So this may be something that affects us in term of airgapp. I actually do not recall the reasons for this being false, but i remember having discussions with @dkoshkin. @dkoshkin could you please give us some input here?

Reply:
Yeah this was done bc of airgap

Reply:
Okay. So then we should build for now the new plugin image. Who can do that @gracedo. Is this already in a CI script?

For the next PR regarding elastic 7.8 we must build and provide hole kibana image with that plugin preinstalled.

Reply:
created PR https://github.com/mesosphere/kubeaddons-sidecars/pull/2

Reply:
Looking at the discussion above, I believe this should stay disabled. Plus, the comment should include a link to https://github.com/mesosphere/kubeaddons-sidecars/ so people know **how** to rebuild the image.